pkg/build/templates/modify-raw-image.sh.tpl

```diff
@@ -13,6 +13,17 @@ set -euo pipefail
 #
 # Guestfish Command Documentation: https://libguestfs.org/guestfish.1.html
 
+# In x86_64, the default root partition is the third partition
+ROOT_PART=/dev/sda3
+
+# Make the necessarry adaptations for aarch64
+if [[ $(uname -m) == "aarch64" ]]; then
+	if ! test -f /dev/kvm; then
+		export LIBGUESTFS_BACKEND_SETTINGS=force_tcg
```

Comment:
I personally believe that this should be the default behavior. Getting KVM to work in a container (in my experience) has been very error prone and while investigating the x86 EIB builds, I found that it resorts to “tcg” anyway so my opinion is that we just make use force_tcg as the default and avoid KVM.